lib/vagrant-openshift/action/install_docker.rb

```diff
@@ -44,7 +44,12 @@ def call(env)
           )
 
           # Configure the Docker daemon
-          sudo(@env[:machine], "SSH_USER='#{ssh_user}' #{home}/configure_docker.sh", :timeout=>60*30)
+          if @options[:skip_volume_group]
```

Comment:
Ah, yeah we may have been running without `nounset`. Ok. I'll let this languish just a bit then, probably get it done late Friday since we'll have to reconfigure all the Jenkins jobs to use the flag.